common/src/main/resources/less/style.less

```diff
@@ -88,6 +84,7 @@ body {
 .explore-tile {
   .full-height-width();
   background: @tileBackground;
+  overflow-y: auto;
```

Comment:
After testing I think this should go to `.explore-tile-body`

Reply:
I originally had it on `.explore-tile-body`. For some reason, the scrolling body still extended about one title bar height below the bottom of the tile. I'll see if I can figure out what bit of CSS was causing that.